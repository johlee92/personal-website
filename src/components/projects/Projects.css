.projects {
    width: 100%;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.projects-message {
    margin: 10px 0;
    width: 80%;
    max-width: 720px;
}

.projects-list {
    margin: 10px 0;
    width: 80%;
    max-width: 720px;
}