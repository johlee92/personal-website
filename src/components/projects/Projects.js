import React, { Component } from 'react';
import './Projects.css';

class Projects extends Component {
    render() {
        return (
            <div className="projects">
                <div className="projects-message">
                    <p>Here's my portfolio</p>
                </div>
                <div className="projects-list">

                </div>
            </div>
        )
    }
}

export default Projects;