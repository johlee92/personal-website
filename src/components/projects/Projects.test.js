import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Projects from './Projects';

describe(`Projects component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <Projects />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <Projects />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
