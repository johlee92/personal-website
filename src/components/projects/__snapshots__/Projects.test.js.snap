// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component renders the component as expected 1`] = `<div />`;
